### PR TITLE
Added comment for clarity

### PR DESCRIPTION
**Pull request by mmayankkkb**
Added timing note for clarity #58

### CODE REVIEW

File: main/main.ino [hunk 1]
```diff
@@ -183,7 +183,8 @@ flightPhase runDescending(int tick){//this runs at 20hz
   return DESCENDING;
 }
 
-flightPhase runPostFlight(int tick){
+flightPhase runPostFlight(int tick){//this runs at 1hz
+
```
Comment (JonathanSafer) [inline main/main.ino:187]: Please remove this extra space on 187

**Review by JonathanSafer — CHANGES_REQUESTED** [review]
Please verify that the program compiles without errors or (additional) warnings

File: main/main.ino (hunk 1 above)
Comment (mmayankkkb) [reply main/main.ino]: Done, please check the recent commit.

**Comment by JonathanSafer** [comment]
Could you provide the output from the compiler log? You can find directions to set up your compilation [here](https://github.com/Harlem-Launch-Alliance/Catalyst-2/blob/main/GETTING_STARTED.md).

**Review by JonathanSafer — CHANGES_REQUESTED** [review]
Please change the title of the PR to `added comment for clarity` or something similar so that we can easily tell what this PR is about

**Comment by mmayankkkb** [comment]
> Could you provide the output from the compiler log? You can find directions to set up your compilation [here](https://github.com/Harlem-Launch-Alliance/Catalyst-2/blob/main/GETTING_STARTED.md).

I tried to verify/compile it , but I got an error in Arduino IDE [grpc: error while marshaling: proto: Marshal called with nil
Compilation error: grpc: error while marshaling: proto: Marshal called with nil] . After researching i got to know that it requires an  Arduino  board to compile and i didn't own it.Project should work fine because I just add an comment.
arduino

**Comment by JonathanSafer** [comment]
This may be because you do not have teensyduino installed. Could you please download the software in the link that I sent and try compiling either in the teensyduino IDE or (preferably) in VSCode with PlatformIO installed? You do not need any hardware to compile the software

**Comment by mmayankkkb** [comment]
> This may be because you do not have teensyduino installed. Could you please download the software in the link that I sent and try compiling either in the teensyduino IDE or (preferably) in VSCode with PlatformIO installed? You do not need any hardware to compile the software

oh.. yeah ..thanks for advice finally i am able to compile it.
                LOGS

Processing teensy41 (platform: teensy; board: teensy41; framework: arduino)
-------------------------------------------------------------------------------------------------------------------------------------------------------------Verbose mode can be enabled via `-v, --verbose` option
CONFIGURATION: https://docs.platformio.org/page/boards/teensy/teensy41.html
PLATFORM: Teensy (4.17.0) > Teensy 4.1
HARDWARE: IMXRT1062 600MHz, 512KB RAM, 7.75MB Flash
DEBUG: Current (jlink) External (jlink)
PACKAGES:
 - framework-arduinoteensy @ 1.157.220801 (1.57)   
 - tool-teensy @ 1.157.0 (1.57)
 - toolchain-gccarmnoneeabi @ 1.50401.190816 (5.4.1)
Converting main.ino
LDF: Library Dependency Finder -> https://bit.ly/configure-pio-ldf
LDF Modes: Finder ~ chain, Compatibility ~ soft
Found 97 compatible libraries
Scanning dependencies...
Dependency Graph
|-- EWMA @ 1.0.2
|-- Adafruit BMP3XX Library @ 2.1.2
|   |-- Adafruit BusIO @ 1.13.2
|   |   |-- Wire @ 1.0
|   |   |-- SPI @ 1.0
|   |-- Wire @ 1.0
|   |-- SPI @ 1.0
|-- Adafruit GPS Library @ 1.7.0
|   |-- SPI @ 1.0
|   |-- Wire @ 1.0
|-- SD @ 2.0.0
|   |-- SdFat @ 2.1.2
|   |   |-- SPI @ 1.0
|-- Wire @ 1.0
Building in debug mode
Compiling .pio\build\teensy41\src\main.ino.cpp.o
Compiling .pio\build\teensy41\lib2f7\EWMA\Ewma.cpp.o
Compiling .pio\build\teensy41\libacf\Wire\Wire.cpp.o
Compiling .pio\build\teensy41\libacf\Wire\WireIMXRT.cpp.o
Compiling .pio\build\teensy41\libacf\Wire\WireKinetis.cpp.o
Compiling .pio\build\teensy41\libacf\Wire\utility\twi.c.o
Compiling .pio\build\teensy41\lib9a4\SPI\SPI.cpp.o
Compiling .pio\build\teensy41\lib745\Adafruit BusIO\Adafruit_BusIO_Register.cpp.o
Compiling .pio\build\teensy41\lib745\Adafruit BusIO\Adafruit_I2CDevice.cpp.o
Compiling .pio\build\teensy41\lib745\Adafruit BusIO\Adafruit_SPIDevice.cpp.o
Compiling .pio\build\teensy41\lib91c\Adafruit BMP3XX Library\Adafruit_BMP3XX.cpp.o
Compiling .pio\build\teensy41\lib91c\Adafruit BMP3XX Library\bmp3.c.o
In file included from C:/Users/mayank/Documents/Git/Catalyst-2/main/main.ino:13:0:
main\gy521_imu.h: In function 'imuReading getIMU()':
main\gy521_imu.h:16:41: warning: variable 'Tmp' set but not used [-Wunused-but-set-variable]
   int16_t AcX, AcY, AcZ, GyX, GyY, GyZ, Tmp;
                                         ^
In file included from C:/Users/mayank/Documents/Git/Catalyst-2/main/main.ino:15:0:
main\gps.h: In function 'String setupGPS()':
main\gps.h:42:23: warning: invalid conversion from 'char*' to 'char' [-fpermissive]
       c = GPS.lastNMEA(); //this gives an error but doesn't seem to be an issue
                       ^
main\gps.h:36:10: warning: variable 'c' set but not used [-Wunused-but-set-variable]        
     char c = GPS.read();
          ^
In file included from C:/Users/mayank/Documents/Git/Catalyst-2/main/main.ino:15:0:
main\gps.h: In function 'gpsReading getGPS()':
main\gps.h:85:23: warning: invalid conversion from 'char*' to 'char' [-fpermissive]
       c = GPS.lastNMEA(); //this gives an error but doesn't seem to be an issue
                       ^
main\gps.h:79:10: warning: variable 'c' set but not used [-Wunused-but-set-variable]
     char c = GPS.read();
          ^
C:/Users/mayank/Documents/Git/Catalyst-2/main/main.ino: In function 'void loop()':
C:/Users/mayank/Documents/Git/Catalyst-2/main/main.ino:59:11: warning: unused variable 'x' [-Wunused-variable]
       int x = 1 + 1;
           ^
C:/Users/mayank/Documents/Git/Catalyst-2/main/main.ino:65:11: warning: unused variable 'x' [-Wunused-variable]
       int x = 1 + 1;
           ^
Archiving .pio\build\teensy41\lib2f7\libEWMA.a
Compiling .pio\build\teensy41\lib26d\Adafruit GPS Library\Adafruit_GPS.cpp.o
Compiling .pio\build\teensy41\lib26d\Adafruit GPS Library\NMEA_build.cpp.o
Compiling .pio\build\teensy41\lib26d\Adafruit GPS Library\NMEA_data.cpp.o
Compiling .pio\build\teensy41\lib26d\Adafruit GPS Library\NMEA_parse.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\ExFatLib\ExFatDbg.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\ExFatLib\ExFatFile.cpp.o
Archiving .pio\build\teensy41\lib745\libAdafruit BusIO.a
Archiving .pio\build\teensy41\libacf\libWire.a
Compiling .pio\build\teensy41\lib931\SdFat\ExFatLib\ExFatFilePrint.cpp.o
Archiving .pio\build\teensy41\lib9a4\libSPI.a
Indexing .pio\build\teensy41\lib2f7\libEWMA.a
Archiving .pio\build\teensy41\lib91c\libAdafruit BMP3XX Library.a
Indexing .pio\build\teensy41\lib745\libAdafruit BusIO.a
Compiling .pio\build\teensy41\lib931\SdFat\ExFatLib\ExFatFileWrite.cpp.o
Indexing .pio\build\teensy41\libacf\libWire.a
Compiling .pio\build\teensy41\lib931\SdFat\ExFatLib\ExFatFormatter.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\ExFatLib\ExFatName.cpp.o
Indexing .pio\build\teensy41\lib9a4\libSPI.a
Archiving .pio\build\teensy41\lib26d\libAdafruit GPS Library.a
Indexing .pio\build\teensy41\lib91c\libAdafruit BMP3XX Library.a
Compiling .pio\build\teensy41\lib931\SdFat\ExFatLib\ExFatPartition.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\ExFatLib\ExFatVolume.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\ExFatLib\upcase.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\FatLib\FatDbg.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\FatLib\FatFile.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\FatLib\FatFileLFN.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\FatLib\FatFilePrint.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\FatLib\FatFileSFN.cpp.o
Indexing .pio\build\teensy41\lib26d\libAdafruit GPS Library.a
Compiling .pio\build\teensy41\lib931\SdFat\FatLib\FatFormatter.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\FatLib\FatName.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\FatLib\FatPartition.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\FatLib\FatVolume.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\FreeStack.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\FsLib\FsFile.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\FsLib\FsNew.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\FsLib\FsVolume.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\MinimumSerial.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\SdCard\SdCardInfo.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\SdCard\SdSpiCard.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\SdCard\SdioTeensy.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\SpiDriver\SdSpiArtemis.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\SpiDriver\SdSpiChipSelect.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\SpiDriver\SdSpiDue.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\SpiDriver\SdSpiESP.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\SpiDriver\SdSpiParticle.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\SpiDriver\SdSpiSTM32.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\SpiDriver\SdSpiSTM32Core.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\SpiDriver\SdSpiTeensy3.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\common\FmtNumber.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\common\FsCache.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\common\FsDateTime.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\common\FsGetPartitionInfo.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\common\FsName.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\common\FsStructs.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\common\FsUtf.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\common\PrintBasic.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\common\upcase.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\iostream\StdioStream.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\iostream\StreamBaseClass.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\iostream\istream.cpp.o
Compiling .pio\build\teensy41\lib931\SdFat\iostream\ostream.cpp.o
Compiling .pio\build\teensy41\lib150\SD\SD.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\AudioStream.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\CrashReport.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\DMAChannel.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\EventResponder.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\HardwareSerial.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\HardwareSerial1.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\HardwareSerial2.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\HardwareSerial3.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\HardwareSerial4.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\HardwareSerial5.cpp.o
Archiving .pio\build\teensy41\lib931\libSdFat.a
Compiling .pio\build\teensy41\FrameworkArduino\HardwareSerial6.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\HardwareSerial7.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\HardwareSerial8.cpp.o
Archiving .pio\build\teensy41\lib150\libSD.a
Compiling .pio\build\teensy41\FrameworkArduino\IPAddress.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\IntervalTimer.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\Print.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\Stream.cpp.o
Indexing .pio\build\teensy41\lib931\libSdFat.a
Compiling .pio\build\teensy41\FrameworkArduino\Time.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\Tone.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\WMath.cpp.o
Indexing .pio\build\teensy41\lib150\libSD.a
Compiling .pio\build\teensy41\FrameworkArduino\WString.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\analog.c.o
Compiling .pio\build\teensy41\FrameworkArduino\bootdata.c.o
Compiling .pio\build\teensy41\FrameworkArduino\clockspeed.c.o
Compiling .pio\build\teensy41\FrameworkArduino\debugprintf.c.o
Compiling .pio\build\teensy41\FrameworkArduino\delay.c.o
Compiling .pio\build\teensy41\FrameworkArduino\digital.c.o
Compiling .pio\build\teensy41\FrameworkArduino\eeprom.c.o
Compiling .pio\build\teensy41\FrameworkArduino\extmem.c.o
Compiling .pio\build\teensy41\FrameworkArduino\fuse.c.o
Compiling .pio\build\teensy41\FrameworkArduino\interrupt.c.o
Compiling .pio\build\teensy41\FrameworkArduino\keylayouts.c.o
Compiling .pio\build\teensy41\FrameworkArduino\main.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\memcpy-armv7m.S.o
Compiling .pio\build\teensy41\FrameworkArduino\memset.S.o
Compiling .pio\build\teensy41\FrameworkArduino\new.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\nonstd.c.o
Compiling .pio\build\teensy41\FrameworkArduino\pwm.c.o
Compiling .pio\build\teensy41\FrameworkArduino\rtc.c.o
Compiling .pio\build\teensy41\FrameworkArduino\serialEvent.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\serialEvent1.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\serialEvent2.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\serialEvent3.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\serialEvent4.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\serialEvent5.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\serialEvent6.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\serialEvent7.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\serialEvent8.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\serialEventUSB1.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\serialEventUSB2.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\sm_alloc_valid.c.o
Compiling .pio\build\teensy41\FrameworkArduino\sm_calloc.c.o
Compiling .pio\build\teensy41\FrameworkArduino\sm_free.c.o
Compiling .pio\build\teensy41\FrameworkArduino\sm_hash.c.o
Compiling .pio\build\teensy41\FrameworkArduino\sm_malloc.c.o
Compiling .pio\build\teensy41\FrameworkArduino\sm_malloc_stats.c.o
Compiling .pio\build\teensy41\FrameworkArduino\sm_pool.c.o
Compiling .pio\build\teensy41\FrameworkArduino\sm_realloc.c.o
Compiling .pio\build\teensy41\FrameworkArduino\sm_realloc_i.c.o
Compiling .pio\build\teensy41\FrameworkArduino\sm_realloc_move.c.o
Compiling .pio\build\teensy41\FrameworkArduino\sm_szalloc.c.o
Compiling .pio\build\teensy41\FrameworkArduino\sm_util.c.o
Compiling .pio\build\teensy41\FrameworkArduino\sm_zalloc.c.o
Compiling .pio\build\teensy41\FrameworkArduino\startup.c.o
Compiling .pio\build\teensy41\FrameworkArduino\tempmon.c.o
Compiling .pio\build\teensy41\FrameworkArduino\usb.c.o
Compiling .pio\build\teensy41\FrameworkArduino\usb_audio.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\usb_desc.c.o
Compiling .pio\build\teensy41\FrameworkArduino\usb_flightsim.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\usb_inst.cpp.o
Compiling .pio\build\teensy41\FrameworkArduino\usb_joystick.c.o
Compiling .pio\build\teensy41\FrameworkArduino\usb_keyboard.c.o
Compiling .pio\build\teensy41\FrameworkArduino\usb_midi.c.o
Compiling .pio\build\teensy41\FrameworkArduino\usb_mouse.c.o
Compiling .pio\build\teensy41\FrameworkArduino\usb_mtp.c.o
Compiling .pio\build\teensy41\FrameworkArduino\usb_rawhid.c.o
Compiling .pio\build\teensy41\FrameworkArduino\usb_seremu.c.o
Compiling .pio\build\teensy41\FrameworkArduino\usb_serial.c.o
Compiling .pio\build\teensy41\FrameworkArduino\usb_serial2.c.o
Compiling .pio\build\teensy41\FrameworkArduino\usb_serial3.c.o
Compiling .pio\build\teensy41\FrameworkArduino\usb_touch.c.o
Compiling .pio\build\teensy41\FrameworkArduino\yield.cpp.o
Archiving .pio\build\teensy41\libFrameworkArduino.a
Indexing .pio\build\teensy41\libFrameworkArduino.a
Linking .pio\build\teensy41\firmware.elf
Checking size .pio\build\teensy41\firmware.elf
Calculating size .pio\build\teensy41\firmware.elf
Building .pio\build\teensy41\firmware.hex
teensy_size: Memory Usage on Teensy 4.1:
teensy_size:   FLASH: code:92976, data:12428, headers:8256   free for files:8012804
teensy_size:    RAM1: variables:378016, code:90136, padding:8168   free for local variables:47968
teensy_size:    RAM2: variables:12384  free for malloc/new:511904
=============================================================== [SUCCESS] Took 37.67 seconds ===============================================================
 *  Terminal will be reused by tasks, press any key to close it.

**Review by JonathanSafer — CHANGES_REQUESTED** [review]
Looks good. Please change the title to something more relevant than `Update main.ino` and then I will approve your change. Also as an FYI, we've decided to limit contributions to this repository to members of the [HLA](hla.nyc) organization. We'll accept this one since you already made your changes, but after this if you'd like to continue to work on this project I recommend that you keep your changes in your fork. I will also update our README to reflect this contribution rule so that others are not confused.

**Review by JonathanSafer — APPROVED** [review]

**Pull request merged**